frontend/src/lib/components/Annotations/AnnotationMarker.tsx

```diff
@@ -91,7 +89,8 @@ export function AnnotationMarker({
     const { user } = useValues(userLogic)
     const { currentTeam } = useValues(teamLogic)
     const { currentOrganization } = useValues(organizationLogic)
-    const { diffType, groupedAnnotations } = useValues(annotationsLogic({ insightNumericId }))
+    // insightAnnotationsLogic must be bound using BindLogic
```

Comment:
Why is that the case? Both interested for myself and for future readers who may thing. Okay... but why? :D

Reply:
Ah, this is just because previously we had a lot of prop drilling to get `insightNumericId` into all uses of `annotationsLogic()`, and thanks to `BindLogic` this is no longer the case, as props are bound once. I clarified the comment, hopefully it will be more helpful.